css/css-animations/parsing/animation-delay-invalid.html

```diff
@@ -14,6 +14,9 @@
 test_invalid_value("animation-delay", "infinite");
 test_invalid_value("animation-delay", "0");
 test_invalid_value("animation-delay", "1s 2s");
+
+test_invalid_value("animation-delay", "initial, -3s");
```

Comment:
These changes seem unrelated to what the commit message says, but fine I guess.

Reply:
See the Safari results for animation-shorthand, transition-shorthand
https://wpt.fyi/results/css/css-animations/parsing/animation-shorthand.html?label=pr_head&max-count=1&pr=19195
https://wpt.fyi/results/css/css-transitions/parsing/transition-shorthand.html?label=pr_head&max-count=1&pr=19195

When Safari expands the animation and transition shorthands, many longhands serialize as ".., initial" or "initial, ...".

I added "-invalid" test cases to verify such values are correctly rejected.